pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/client/colorpicker/Color.java

```diff
@@ -516,7 +516,7 @@ public int getRed() {
    * @return green component
    */
   public int getGreen() {
-    return (int) ( g * 255 );
+    return (int) ( g * 255.0f + 0.5f );
```

Comment:
no, it will return 255 - int cast works like floor().
this algorithm comes from java.awt.Color#HSBtoRGB - without it we will have not correct colors after color convert. My tests will fail without that piece of code.